### ISSUE TITLE
On Ubuntu 22.04.1 LTS I cannot install libobs-dev!

### ISSUE THREAD

**Issue by p0o0uya** [royshil/obs-backgroundremoval#111]
I already have obs-studio installed on My Jammy machine and its running well. Now, when I try to install obs-backgroundremoval plugin I encounter a conflict in installation. When I try to install libobs-dev as a requirement I get a conflict error like the following: 
`The following packages have unmet dependencies:
 obs-studio : Conflicts: libobs0 but 27.2.3+dfsg1-1 is to be installed
E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
`
I tried to skip installing this package because I thought it might already have been installed with obs-studio. But, in a later stage when trying to build and install this repository I get the following error which I think regards back to the package above. The error is:
`-- Found Git: /usr/bin/git (found version "2.34.1") 
-- The C compiler identification is GNU 11.2.0
-- The CXX compiler identification is GNU 11.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:106 (find_package):
  By not providing "Findlibobs.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "libobs", but
  CMake did not find one.

  Could not find a package configuration file provided by "libobs" with any
  of the following names:

    libobsConfig.cmake
    libobs-config.cmake

  Add the installation prefix of "libobs" to CMAKE_PREFIX_PATH or set
  "libobs_DIR" to a directory containing one of the above files.  If "libobs"
  provides a separate development package or SDK, be sure it has been
  installed.

-- Configuring incomplete, errors occurred!
`

IS this a compatibility issue or am I wrong somewhere in the middle of the installation process?

**Comment by RobotCaleb** [comment]
There are all sorts of casing issues going on in here. I don't know how this was built by anyone for Ubuntu out of the box.

**Comment by RobotCaleb** [comment]
I got it to build (but haven't tested it) on Ubuntu 22.04 with the following files:
https://gist.github.com/RobotCaleb/6ea4066f220ac8fe5e538e741a873174

Determining the diff is left as an exercise for the reader.

**Comment by BriegerCA** [comment]
Thank you Caleb!!
I finally got it to work with your Code. Was about time!
Granted, the CPU goes mountain climbing, but that was to be expected.
I had to use sudo aptitude install libobs and down grade OBS to compile (why the heck? KDENLIVE left the house as well),
then got the segmentation error. Did the upgrade back to version OBS 28 on UBUNTU 22.04.
Now I am in business.

Thanks again for your Code!!!
